### PR TITLE
LIVE-3075: render links as text in overlay

### PR DESCRIPTION
**Pull request by joecowton1**
## What does this change?
Handles `a` tags in header image overlay.

## Why?
Previously we weren't rendering these so any links were rendered as an empty string.

### Before
Screenshot 2021-09-20 at 12 39 20

### Afte
Screenshot 2021-09-20 at 14 38 35
r

### CODE REVIEW

File: apps-rendering/src/components/editions/headerImageCaption.tsx [hunk 1]
```diff
@@ -85,6 +85,8 @@ const toReact = (node: Node, key: number): ReactNode => {
 	switch (node.nodeName) {
 		case 'EM':
 			return <em>{node.textContent}</em>;
+		case 'A':
+			return <span>{node.textContent ?? ''}</span>;
```
Comment (abeddow91) [inline apps-rendering/src/components/editions/headerImageCaption.tsx:89]: What do we get from the empty string here? If its needed here, should we add it to the EM switch as well?

Comment (joecowton1) [reply apps-rendering/src/components/editions/headerImageCaption.tsx]: Not much tbh, was just copying from another implementation but it works fine without.

**Review by abeddow91 — APPROVED** [review]
Looks good!

**Review by groakland — APPROVED** [review]
LGTM

**Pull request merged**